### PR TITLE
style: update multiple choice input node style

### LINKED ISSUE #1907: Multiple ChoicInput
Hello,

Just another spacing issue on this node, when it add 3 items to a ListType - spacing around the boxes needs to change. I have spec'd out the spacing on all of the node, and it's options. 

Should be a pretty easy fix. 

Thanks again.

Screen Shot 2020-01-27 at 10 52 37 AM

MacOS 10.15.2
Chrome
Stable

### PR DESCRIPTION
**Pull request by alanlong9278**
## Description
UI about multiple choice input  align with design.

## Task Item
closes #1907 

## Screenshots

image

image

### CODE REVIEW

**Comment by cwhitten** [comment]
Hi @alanlong9278 

I believe we need to add a border radius to the think blue border on the top node edges. cc @DesignPolice

**Comment by DesignPolice** [comment]
Hey Alan,

not sure why I didn't get an email on this - I need to check my email settings. Sorry to not get back to you sooner.

I have a couple of current issues that also have I have noticed a similar treatment, I would love to just make a couple of tweaks.

The top corners on the nodes have a corner radius of 2px - the hoover outline should have that as well.

I have mocked up some screen shots on these two. 

[ChoiceInput Node Edit #1905](https://github.com/microsoft/BotFramework-Composer/issues/1905) - these will need changes on other nodes

Hoover State on Nodes #1904 [https://github.com/microsoft/BotFramework-Composer/issues/1904](url)

but also attached here.

reach out if that you would like more clarity... happy to help out. Thank you!
M

screen grab

File: Composer/packages/extensions/visual-designer/src/components/nodes/steps/ChoiceInput.tsx [hunk 1]
```diff
@@ -21,7 +21,7 @@ export const ChoiceInputChoices = ({ choices }) => {
   }
 
   return (
-    <div data-testid="ChoiceInput" css={{ padding: '0 0 8px 45px' }}>
+    <div data-testid="ChoiceInput" css={{ padding: '0 0 16px 29px' }}>
```
Comment (yeze322) [inline Composer/packages/extensions/visual-designer/src/components/nodes/steps/ChoiceInput.tsx]: can this value be calculated from some constants?

Comment (DesignPolice) [reply Composer/packages/extensions/visual-designer/src/components/nodes/steps/ChoiceInput.tsx]: @yeze322 I'm not sure how to answer that... the goal is that ideally things are all multiples of 4. 

I'm not a math guy, but happy to try and help if there is some sort of ratio you want to make that seems like a good idea... or if switching to SVG files makes life easier for you?

Comment (yeze322) [reply Composer/packages/extensions/visual-designer/src/components/nodes/steps/ChoiceInput.tsx]: @DesignPolice Thanks Marc! I think your design has already covered those pixel numbers. Programmatically, we need some code optimization to align to your design.

**Pull request merged**